indico/modules/events/tracks/templates/_track_list.html

```diff
@@ -1,70 +1,125 @@
+                    {%- trans title=track.default_session.title -%}
+                        Indico will preselect the session "{{ title }}" whenever an abstract is accepted for
+                        this track.
+                    {%- endtrans -%}
+                {% endset %}
+                <span class="i-label small default-session"
+                      title='{{ default_session_tooltip }}'
+                      style="{{ track.default_session.colors.css }}">
+                    {{ track.default_session.title }}
+                </span>
+            {% endif %}
+        </div>
+        {% if track.description %}
+            <div class="track-content i-box-content">
+                {{ track.description }}
+            </div>
+        {% endif %}
+    </li>
+{% endmacro %}
+
+{% macro render_group(group, event) %}
+    <li class="track-row i-box track-group-box" data-id="{{ group.id }}">
+        <div class="ui-i-box-sortable-handle track-group-handle hide-if-locked"></div>
+        <div class="i-box-header">
+            <span class="i-box-title">{{ group.title }}</span>
+            <div class="i-box-buttons hide-if-locked">
+                <div class="toolbar">
+                    <div class="group">
+                        <a href="#" class="i-button icon-edit js-edit-item"
+                           title="{% trans %}Edit track group{% endtrans %}"
+                           data-href="{{ url_for('tracks.edit_track_group', group) }}"
+                           data-title='{% trans title=group.title %}Edit track group "{{ title }}"{% endtrans %}'
+                           data-update="#track-list-container"
+                           data-ajax-dialog></a>
+                    </div>
+                    <div class="group">
+                        <a href="#" class="i-button danger text-color icon-remove js-delete-item"
+                           title="{% trans %}Delete track group{% endtrans %}"
+                           data-href="{{ url_for('tracks.delete_track_group', group) }}"
+                           data-confirm="{% trans %}Are you sure you want to delete this track group?{% endtrans %}"
+                           data-title='{% trans title=group.title %}Delete track "{{ title }}"{% endtrans %}'
+                           data-method="DELETE"
+                           data-update="#track-list-container"></a>
+                    </div>
+                </div>
+            </div>
+        </div>
+        {% if group.description %}
+            <div class="track-content i-box-content track-group-box-content">
+                {{ group.description }}
+            </div>
+        {% endif %}
+            <div  class="embedded-track-list">
+                <ul class="i-box-group vert track-list" data-url="{{ url_for('.sort_tracks', event) }}">
+                    <div class="no-tracks-message">
+                        <i>No tracks have been added yet.</i>
```

Comment:
Use `<em>`. The `<i>` element is not really used anymore for (italic/emphasized) styling anymore, just like one uses `<strong>` instead of `<b>`.